app/src/main/java/com/ispc/gymapp/options/Mi_Perfil.java

```diff
@@ -0,0 +1,23 @@
+package com.ispc.gymapp.options;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+
+import com.ispc.gymapp.R;
+
+public class Mi_Perfil extends AppCompatActivity {
```

Comment:
```suggestion
public class MiPerfil extends AppCompatActivity {
```
por convención en Java las clases y los métodos no se escriben con guiones, solo es valido para las constantes